### DISCUSSION TITLE
Add `aiida` as CLI command

### DISCUSSION THREAD

**Discussion by edan-bainglass** [aiidateam/aiida-core#6381] [Ideas]
Per @giovannipizzi, `aiida` was considered originally as the CLI tool but clashed with the similarly named library, hence `verdi` was born. However, this appears to no longer be an issue (tested). I then propose to replace `verdi` with `aiida`. I argue that it would be difficult to argue against `aiida` being a clearer choice. `verdi` can still exist for legacy purposes, but perhaps for new users, `aiida` is the way? We could update our language everywhere to reflect this.

**Comment by mbercx** [comment]
<img src="https://github.com/aiidateam/aiida-core/assets/25607960/fc1bcfa0-cbb1-46e3-b606-fa89bc16ee2f" alt="drawing" width="400"/>

**Reply by mbercx** [reply]
More seriously, I think having an additional `aiida` entry point to the CLI makes plenty of sense. Also don't see any objections to making this the default in the documentation. :)

**Reply by edan-bainglass** [reply]
Thanks @mbercx. I agree regarding the docs. I'll bring it up next AiiDA meeting. Sorry Verdi.

**Comment by sphuber** [comment]
To give some counterweight to this proposal: does the fact that the CLI is called `verdi` instead of `aiida` really cause any problems? I have only ever heard one person who thought it was a weird choice and found it counterintuitive. On the flipside, I think there is actually a potential benefit to having `verdi` instead of `aiida`. The name `verdi` provides a unique label to refer to the CLI. When talking about it, it is easy to refer to the CLI and everybody knows what you are talking about. If we were to change this for `aiida`, I can see potentially discussions becoming less obvious. Now when someone refers to "aiida", you are not immediately sure if it refers to the software as a whole, the Python API, or the CLI.

So, considering the, what I think, minimal advantages of switching to `aiida`, and the considerable amount of work for us and users to update all their scripts/docs, combined with the downside of  introducing ambiguity, is this change worth it?

**Comment by sphuber** [comment]
Should this be converted to a feature-request issue instead perhaps?

**Comment by edan-bainglass** [comment]
Yes? We have perhaps one too many avenues for discussion. Note that @danielhollas mentioned we might want to Discourse this, which I'm fine with. I suppose we can do both?

**Comment by danielhollas** [comment]
The whole idea of Discourse was to centralize discussions. For feature requests that require extended discussions I'd say Discourse is better than GitHub issues.

@sphuber I think we should just close / lock the GitHub Discussions feature.

**Comment by sphuber** [comment]
Converting this to an issue was just to formally track it in Github. I am working on opening a Discourse discussion as that should indeed be the main venue for design discussions.

**Comment by sphuber** [comment]
I opened the discussion here: https://aiida.discourse.group/t/should-verdi-be-renamed-to-aiida-or-add-aiida-as-an-alias/406
